storage/unit_tests/test_bucket.py

```diff
@@ -1068,6 +1084,75 @@ def dummy_response():
         self.assertEqual(page2.num_items, 0)
         self.assertEqual(iterator.prefixes, set(['foo', 'bar']))
 
+    def _test_generate_upload_policy_helper(self, **kwargs):
+        import base64
+        import json
+
+        credentials = _create_signing_credentials()
+        credentials.signer_email = mock.sentinel.signer_email
```

Comment:
Prefer a tighter spec. Maybe create your own credential here with a fixed spec? Or does it have one already?